src/config.rs

```diff
@@ -259,6 +262,26 @@ macro_rules! create_config {
             )+
         }
 
+        pub struct ExperimentalFeature<'a>(&'a Config);
+
+        impl<'a> ExperimentalFeature<'a> {
+            $(
+            pub fn $i(&self) -> bool {
+                (self.0).$i.3
+            }
+            )+
+        }
+
+        pub struct ExperimentalFeatureSetter<'a>(&'a mut Config);
+
+        impl<'a> ExperimentalFeatureSetter<'a> {
+            $(
+            pub fn $i(&mut self, value: bool) {
+                (self.0).$i.3 = value;
+            }
+            )+
+        }
+
```

Comment:
I don't think we need to be able to set whether each option is stable or unstable, so I don't think we need these getter/setters. A client only needs to be able to set the unstable-features flag itself, which I think it can do with the existing setters.

Reply:
I implemented this setter/getter, considering this
>> (we would need some mechanism for marking options as stable or unstable)"

Reply:
That is covered where you added a flag to each option for whether it is stable or unstable.